code/formatters.py

```diff
@@ -263,6 +263,9 @@ def formatters_reformat_last(formatters: str) -> str:
     def formatters_reformat_selection(formatters: str) -> str:
         """Reformats the current selection."""
         selected = edit.selected_text()
+        if not selected:
+            print("Asked to reformat selection, but nothing selected!")
```

Comment:
We don't have a warning system to use for this kind of thing? If not then obv out of scope here but might be a useful thing to make at some point

Reply:
there is Python's logging.warning() but @knausj85 seemed to prefer print()?

Reply:
I'm fine with logging.warning - my suggestion on another PR was to open an issue so we remember switch to logging.warning more broadly if its useful to folks. 

I previously hadn't really bothered since I run with zero output unless explicitly debugging something.